tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -48,4 +48,18 @@ sub run {
     $self->reboot_and_wait_up($timeout);
 }
 
+sub enable_y2log_analysis {
+    # Create and Upload y2log for analysis
```

Comment:
If this is going to be called by $self->xxxx, I think you miss my $self = shift; here ?

Reply:
Good point, had changed as $self = shift;

Reply:
DONE